website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step08-apply-helm-chart.sh

```diff
@@ -1,6 +1,9 @@
 # Logout of helm registry to perform an unauthenticated pull against the public ECR
```

Comment:
Also, if you are making this change, can you also make sure that you make the change on the `/preview` path since this is the path that we take forward when going from version to version of Karpenter?

Reply:
Done